nservicebus/hosting/cloud-services-host/index.md

```diff
@@ -37,6 +36,8 @@ snippet: AzureServiceBusTransportWithAzureHost
 
 Partial: Worker
 
+When self-hosting everything can be configured using code API and extension methods available in the NServiceBus Azure related packages. In such a case it's not required to reference the hosting package in the project. To self-host and endpoint add the required configuration to the role entry point. For more information how to self-host in cloud services refer to the cloud services [self-hosting sample](/samples/azure/self-host/sample.md).
```

Comment:
> To self-host and endpoint add the 

Should be "To self-host **an** endpoint **,** add the..."

Reply:
>how to self-host in cloud services refer to the cloud services

"how to self-host in **C**loud **S**ervices refer to the **C**loud **S**ervices"